runtime/binding-kafka-grpc/src/main/java/io/aklivity/zilla/runtime/binding/kafka/grpc/internal/stream/KafkaGrpcRemoteServerFactory.java

```diff
@@ -1956,6 +1985,7 @@ private MessageConsumer newKafkaFetch(
                 .typeId(kafkaTypeId)
                 .merged(m -> m.capabilities(c -> c.set(FETCH_ONLY))
                     .topic(condition.topic())
+                    .groupId("zilla-kafka-grpc")
```

Comment:
This needs to be configurable, I'm thinking as part of `KafkaConfiguration`, agree?